code/modules/goonchat/browserassets/css/browserOutput_white.css

```diff
@@ -407,6 +407,20 @@ h1.alert, h2.alert		{color: #000000;}
 	100%	{color: #3bb5d3;}
 }
 
+.rainbow				{color: #660015; 	font-weight: bold; animation: rainbowanim 5000ms infinite;}
+.bigrainbow				{color: #660015; 	font-weight: bold;	font-size: 185%; animation: rainbowanim, 5000ms infinite;}
+	@keyframes rainbowanim {
+	0% { color: #FF0000; }
+	13% { color: #FFA321; }
+	26% { color: #F6FF00; }
+	39% { color: #09B800; }
+	52% { color: #00BFE0; }
+	65% { color: #4059FF; }
+	78% { color: #9900FF; }
+	91% { color: #FF0099; }
+	100% { color: #FF0066; }
+}
```

Comment:
This will look at bit odd at the end as it jumps from FF0066 to FF0000, should have the last colour same as the first one.